src/plugins/crossref_event_data/crossref_event_data.py

```diff
@@ -76,6 +77,10 @@ def _build(self, event_data, uri_id, origin, event_dict):
                 )
                 event_dict.update(subj=event)
 
+            existing_raw_ids = check_existing_entries(
+                RawEvent.external_id,
+                [entry['external_id'] for entry in event_list]
```

Comment:
Not sure where this `event_list` is coming from, but propagating it as a queryset might give us more filtering capability, not limited to a simple iteration.

Reply:
`event_list` contains Crossref Event Data data just after we have retrieved it from the API, so we can't use it as a queryset just yet, unfortunately.